Microsoft.Toolkit.Uwp.SampleApp/Pages/About.xaml

```diff
@@ -4,7 +4,7 @@
       xmlns:animations="using:Microsoft.Toolkit.Uwp.UI.Animations"
       xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
       xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-      xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI.Extensions"
+      xmlns:uwpui="using:Microsoft.Toolkit.Uwp.UI"
```

Comment:
I think `uwpui` was used elsewhere in one of the controls, but just thoughts on using `ui` for short as the namespace? As I don't think we'd have anything from the `uwp` package itself which would ever be relevant in the XAML layer, that's part of the point of the 'UI' package. 🙂 Thoughts?

Part of the reason we had `ex` everywhere was because it was short and all our `*Extensions` names are long 😋

Reply:
Yup `ui` makes perfect sense - the base package could just use `uwp` anyway in case it did have other extensions 😄
Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3743/commits/9de627a943d7c507e60ee63ea216ab2ac28a7e97.